corehq/apps/users/templates/users/partials/edit_role_modal.html

```diff
@@ -487,6 +487,44 @@ <h4 class="modal-title" data-bind="text: modalTitle"></h4>
                 </div>
               </div>
             </div>
+            <div class="form-group">
+              <label class="col-sm-4 control-label">
+                {% trans "Assign All Profiles" %}
```

Comment:
nit: thoughts on specifying this as "User Profiles"?

Reply:
I tend to agree that "Assign All User Profiles" would be more clear. The same would go for "Assign Specific User Profiles". This wording is taken from the design doc (I'll link it in the description), so I'll leave a note there for Mruga about it.